aws_lambda_powertools/logging/types.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Dict, Iterable, Union
+
+from typing_extensions import TypedDict
+
+class TypedLog(TypedDict, total=False):
+    level: Union[str, int]
+    location: str
+    timestamp: Union[str, int]
```

Comment:
```suggestion
    timestamp: str | int
```

see how much nicer it becomes after you bring in `from __future__ import annotations` I suggested earlier?